js/apps/admin-ui/src/realm-settings/EmailTab.tsx

```diff
@@ -33,18 +33,20 @@ import "./realm-settings-section.css";
 
 type RealmSettingsEmailTabProps = {
   realm: RealmRepresentation;
+  save: (realm: RealmRepresentation) => void;
 };
 
 export const RealmSettingsEmailTab = ({
   realm: initialRealm,
```

Comment:
Since this is the only place we're getting the realm from there is no longer a need to keep calling it `initialRealm`.
```suggestion
  realm,
```